config.yml

```diff
@@ -0,0 +1,79 @@
+# Acoustic Multiclass Training config file
+
+# Required path related config
+dataframe_csv:
+data_path:
+
+# Dataframe column names
+offset_col: "OFFSET"
+duration_col: "DURATION"
+file_name_col: "FILE NAME"
+manual_id_col: "MANUAL ID"
+
+# Training params
+train_batch_size: 4
+validation_batch_size: 4
+jobs: 2
+logging: true
```

Comment:
changing this to false doesn't turn off logging

Reply:
Did you change it in personal_config.yml?? @sprestrelski

Reply:
sorry it's `config_personal.yml`

Reply:
ok nevermind that is a bug 💀

Reply:
Ah its because of the args parser thingy

Reply:
It must be overwriting it?

Reply:
One sec I can fix this

Reply:
Ok the whole disable logging command line argument was overwriting whether logging was actually enabled. So I had to change it so you'd have to type either --logging False or --logging True @Sean1572

Reply:
> Ok the whole disable logging command line argument was overwriting whether logging was actually enabled. So I had to change it so you'd have to type either --logging False or --logging True @Sean1572

I added some code to check if the default value is the same in the config and the flag. if it is then enable the argument parsing

Reply:
I fixed sean's code from not working